### PR TITLE
Rawkode bugfix/windows phone user agent detection

### PR DESCRIPTION
**Pull request by lsmith77**

### CODE REVIEW

**Comment by lsmith77** [comment]
supersedes https://github.com/liip/LiipThemeBundle/pull/111

File: .travis.yml [hunk 1]
```diff
@@ -35,4 +25,15 @@ matrix:
...
       env: SYMFONY_VERSION=2.5.*
     - php: 5.5
-      env: SYMFONY_VERSION=dev-master
+      env: SYMFONY_VERSION=2.6.*
+    - php: 5.5
+      env: SYMFONY_VERSION='2.7.*@dev'
+
+before_script:
+  - composer self-update
+  - composer require symfony/framework-bundle:${SYMFONY_VERSION}
```
Comment (stof) [inline .travis.yml]: this does not actually test against Symfony 2.3, given that Frameworkbundle 2.3 is compatible with HttpFoundation 2.7 for instance

File: composer.json [hunk 2]
```diff
@@ -20,7 +20,8 @@
         "symfony/framework-bundle": "~2.0"
     },
     "require-dev":{
-        "symfony/console": "~2.0"
+        "symfony/console": "~2.0",
+        "symfony/expression-language": "~2.6"
```
Comment (stof) [inline composer.json]: this will forbid installing a fullstack 2.4 for testing

Comment (lsmith77) [inline composer.json]: I checked and at least for 2.3 it seemed to install FrameworkBundle 2.3 .. wasn't sure if there is a more elegant way to handle the issue.

**Pull request merged**